app/components/common/navbar/big_navbar/NavBar.js

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Drawer from "@mui/material/Drawer";
+import Button from "@mui/material/Button";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import SmallFooter from "../SmallFooter";
+import NAVBAR_LINKS from "@/components/lists/navbar_links";
+import Link from "next/link";
+import Image from "next/image";
+
+export default function NavBarBig() {
+  const [state, setState] = React.useState({
+    top: false,
+    bottom: false,
+    menu: true,
+  });
+
+  const toggleDrawer = (anchor, open) => (event) => {
+    if (
+      event.type === "keydown" &&
+      (event.key === "Tab" || event.key === "Shift")
+    ) {
+      return;
+    }
+
+    setState({ ...state, [anchor]: open });
+  };
+
+  const list = (anchor) => (
```

Comment:
is there a reason for this to not be in a separate file?

Reply:
this component was removed from a UI component's framework.
So, for easier maintenance (if the component is updated in the framework) will be easier to update as well in this project